staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/start.go

```diff
@@ -58,7 +60,7 @@ func DefaultServerConfig() (*extensionsapiserver.Config, error) {
 	if err := options.RecommendedOptions.SecureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
 		return nil, fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
-	if err := options.RecommendedOptions.ApplyTo(genericConfig); err != nil {
+	if err := options.RecommendedOptions.ApplyTo(genericConfig, runtime.NewScheme(), nil); err != nil {
```

Comment:
same here, we have a scheme for the apiextensions-apiserver somewhere.

Reply:
I see.